stylelint.config.js

```diff
@@ -56,8 +56,8 @@ module.exports = {
     "selector-max-compound-selectors": 4,
     // "id,class,type"
     "selector-max-specificity": "1,3,3",
-    "selector-no-id": true,
-    "selector-no-universal": true,
+    "selector-max-id": 0,
+    "selector-max-universal": 0,
```

Comment:
i'd argue for allowing `1` universal selector to support `[attribute]` selectors. 
for instance, blueprint disables this rule to allow such things.